srcpkgs/sile/template

```diff
@@ -1,27 +1,40 @@
 # Template file for 'sile'
 pkgname=sile
-version=0.9.5.1
-revision=9
+version=0.15.5
+revision=1
 build_style=gnu-configure
-hostmakedepends="pkg-config"
+build_helper=rust
+configure_args="--with-system-lua-sources --with-system-luarocks"
+hostmakedepends="jq poppler cargo rust pkg-config"
 makedepends="harfbuzz-devel lua51-devel lua51-lpeg lua51-luaexpat
- lua51-zlib lua51-luafilesystem lua51-luasocket lua51-luasec"
-depends="lua51-lpeg lua51-luaexpat lua51-zlib lua51-luafilesystem
- lua51-luasocket lua51-luasec"
+ lua51-zlib lua51-luafilesystem lua51-luasocket lua51-luasec lua51-cassowary
+ lua51-compat53 lua51-cldr lua51-fluent lua51-loadkit lua51-linenoise
+ lua51-cliargs lua51-luarepl lua51-luasocket lua51-luasec lua51-luautf8
+ lua51-vstruct fontconfig-devel LuaJIT-devel font-sil-gentium rust-std"
+depends="LuaJIT lua51-lpeg lua51-luaexpat lua51-zlib lua51-luafilesystem lua51-cassowary
+ lua51-luasocket lua51-luasec lua51-compat53 lua51-cldr lua51-fluent lua51-loadkit
+ lua51-linenoise lua51-cliargs lua51-luarepl lua51-luasocket lua51-luasec lua51-luautf8
+ lua51-vstruct font-sil-gentium"
 short_desc="Modern typesetting system inspired by TeX"
 maintainer="John <me@johnnynator.dev>"
 license="MIT"
 homepage="http://www.sile-typesetter.org/"
-distfiles="https://github.com/simoncozens/sile/releases/download/v${version}/sile-${version}.tar.bz2"
-checksum=60cdcc4509971973feab352dfc1a86217cc1fdb12d56823f04d863afef92003a
+distfiles="https://github.com/simoncozens/sile/releases/download/v${version}/sile-${version}.tar.zst"
```

Comment:
```suggestion
distfiles="https://github.com/sile-typesetter/sile/releases/download/v${version}/sile-${version}.tar.zst"
```

The canonical URL is the org and has been for a long time. Simon's namespace still has a forward, but that would break if he were to ever create a fork of his own to work on.

Reply:
fixed